src/components/pages/RegisterPage.vue

```diff
@@ -0,0 +1,316 @@
+                            <input 
+                                id="password" 
+                                type="password"
+                                name="password"
+                                class="form-input" 
+                                placeholder="Password" 
+                                v-bind:class="{
+                                    'is-valid' : validation.valid.password,
+                                    'is-invalid' : validation.invalid.password
+                                }"
+                                v-on:focus="validation('password')"
+                                v-model="password"
+                                required
+                            />
+                        </div>
+
+                        <!--- Invalid password confirmation -->
+                        <span class="invalid-feedback" v-if="errorMessagePW">
+                            {{ errorMessagePW }}
+                        </span>
+                        <div class="form-field">
+                            <label class="lock-confirm" for="login-password"><span class="hidden">Confirm Password</span></label>
+                            <input 
+                                id="login-password" 
+                                type="password" 
+                                class="form-input" 
+                                placeholder="Confirm Password" 
+                                v-model="passwordVerify"
+                                @input="validateFormPW"
+                                required
+                            />
+                        </div>
+                            
+                        <div class="form-field">
+                            <input type="submit" value="Register Now">
+                        </div>
+                        <p class="register-link">Have already an account!
+                            <router-link to="/login"><span class="link">Login here</span></router-link>
+                        </p>
+                    </form>
+                </div>
+            </div>
+        </LayoutDiv>
+    </div>
+</template>
+
+<script>
+import CustomHeader from '@/components/shared/CustomHeader.vue';
+import NavBar from '@/components/shared/NavBar.vue';
+import LayoutDiv from '@/components/shared/LayoutDiv.vue';
+
+export default {
+    components: {
+        CustomHeader,
+        NavBar,
+        LayoutDiv
+    },
+    data() {
+        return {
+            email: '',
+            emailBlured: false,
+            emailVerify: '',
+            password: '',
+            passwordVerify: '',
+            passwordBlured: false,
+            valid: false,
+            submitted: false,
+            errorMessageEM: '',
+            errorMessagePW: '',
+
+            validation: {
+                invalid: {},
+                valid: {}
+            }
+        };
+    },
+
+    methods: {
```

Comment:
You need to add a method for what happens when the user clicks on the Submit button. The method should verify that all of the inputs have values. If all input fields are filled out, create an object that will be the payload when we have a registration endpoint in the API. For now, just console.log a message saying 'Submitting registration' and console.log the payload object.